.github/workflows/scale-test-egw.yaml

```diff
@@ -276,15 +283,56 @@ jobs:
           sparse-checkout: clusterloader2
           path: perf-tests
 
-      - name: Generate SSH keypair
+      - name: Create EKS cluster
+        id: deploy-cluster
+        uses: ./.github/actions/setup-eks-cluster
+        with:
+          cluster_name: ${{ steps.vars.outputs.cluster_name }}
+          region: ${{ steps.vars.outputs.eks_region }}
+          owner: "${{ steps.vars.outputs.owner }}"
+          version: "${{ steps.vars.outputs.eks_version }}"
+          addons: "coredns"
+
+      - name: Install Cilium CLI
+        uses: cilium/cilium-cli@1432ef0e4e0cd766f5c1ee0544759d7a31cd32b2 # v0.17.0
+        with:
+          skip-build: ${{ env.CILIUM_CLI_SKIP_BUILD }}
+          image-repo: ${{ env.CILIUM_CLI_IMAGE_REPO }}
+          image-tag: ${{ inputs.SHA || github.sha }}
```

Comment:
Shouldn't that be:
```
image-tag: ${{ steps.vars.outputs.sha }}
```
?

Reply:
That's actually equivalent, because `${{ steps.vars.outputs.sha }}` defaults to `inputs.SHA` for `workflow_dispatch` events (the only that set that variable), and `github.sha` for all the others.